schema.json

```diff
@@ -0,0 +1,140 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/cloudquery/filetypes/v4/file-spec",
+  "$ref": "#/$defs/FileSpec",
+  "$defs": {
+    "FileSpec": {
+      "$id": "/schemas/FileSpec",
+      "$defs": {
+        "CSVSpec": {
+          "properties": {
+            "skip_header": {
+              "type": "boolean",
+              "description": "Specifies if the first line of a file should be the header.",
+              "default": false
+            },
+            "delimiter": {
+              "type": "string",
+              "pattern": "^.$",
+              "description": "Character that will be used as the delimiter.",
+              "default": ","
+            }
+          },
+          "additionalProperties": false,
+          "type": "object",
+          "description": "CloudQuery CSV file output spec."
+        },
+        "JSONSpec": {
```

Comment:
The spec for `JSON` and `Parquet` is an empty object?

Reply:
yes